ansible/roles/common/tasks/fedora-install.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: set package manager
+  set_fact:
+    ansible_pkg_mgr: dnf
+  when: ansible_distribution_major_version|int >= 22
```

Comment:
No. I'll back this off... it is a bug that i need to patch in my local sandbox only.